src/components/icon-button/icon-button.scss

```diff
@@ -85,6 +85,43 @@ $icon-button-padding: 4px;
   }
 }
 
+@mixin icon-button-secondary-white {
+  background: transparent;
+  color: $color-snow-white;
+
+  &:hover,
```

Comment:
You may add the block `&:not(&--disabled)` here. If you do that, you don't need to add the behaviors when it's disabled, like you are doing below.